sensitivity-analysis-api/src/main/java/com/powsybl/sensitivity/SensitivityAnalysisResult.java

```diff
@@ -195,44 +195,44 @@ public boolean contingenciesArePresent() {
      * Get a collection of all the sensitivity values associated with given function
      * for a specific contingency.
      *
-     * @param function sensitivity function
+     * @param function sensitivity function type
      * @param contingencyId the ID of the considered contingency
      * @return a collection of all the sensitivity values associated with given function for the given contingencyId
      */
-    public Collection<SensitivityValue> getSensitivityValuesByFunction(SensitivityFunction function, String contingencyId) {
+    public Collection<SensitivityValue> getSensitivityValuesByFunction(SensitivityFunctionType function, String contingencyId) {
         return sensitivityValuesContingencies.get(contingencyId).stream()
-                .filter(sensitivityValue -> sensitivityValue.getFactor().getFunction().equals(function))
+                .filter(sensitivityValue -> sensitivityValue.getFactor().getFunctionType().equals(function))
                 .collect(Collectors.toList());
     }
 
     /**
      * Get a collection of all the sensitivity values associated with given variable
      * for a specific contingency.
      *
-     * @param variable sensitivity variable
+     * @param variable sensitivity variable type
      * @param contingencyId the ID of the considered contingency
      * @return a collection of all the sensitivity values associated with given variable
      */
-    public Collection<SensitivityValue> getSensitivityValuesByVariable(SensitivityVariable variable, String contingencyId) {
+    public Collection<SensitivityValue> getSensitivityValuesByVariable(SensitivityVariableType variable, String contingencyId) {
         return sensitivityValuesContingencies.get(contingencyId).stream()
-                .filter(sensitivityValue -> sensitivityValue.getFactor().getVariable().equals(variable))
+                .filter(sensitivityValue -> sensitivityValue.getFactor().getVariableType().equals(variable))
                 .collect(Collectors.toList());
     }
 
     /**
      * Get the sensitivity value associated with given function and given variable for a specific contingency.
      *
-     * @param function sensitivity function
-     * @param variable sensitivity variable
+     * @param function sensitivity function type
+     * @param variable sensitivity variable type
      * @param contingencyId the ID of the considered contingency
      * @return the sensitivity value associated with given function and given variable
      */
-    public SensitivityValue getSensitivityValue(SensitivityFunction function, SensitivityVariable variable, String contingencyId) {
+    public SensitivityValue getSensitivityValue(SensitivityFunctionType function, SensitivityVariableType variable, String contingencyId) {
```

Comment:
This does not make sens anymore this kind of method with new api, look at the one in OLF which is more adapted.